website/docs/docs/guides/understanding-state.md

```diff
@@ -51,10 +51,21 @@ After issuing one of the above commands, you can reference the results by adding
 $ dbt run --select result:<status> --defer --state path/to/prod/artifacts
 ```
 
+The available options depend on the node type: 
+
+|                | model | seed | snapshot | test |
+|----------------|-------|------|------|----------|
+| `result:error`   | ✅    | ✅    | ✅    |  ✅      |
+| `result:success` | ✅    | ✅    | ✅     |         |
+| `result:skipped` | ✅    |      |  ✅    |        |
```

Comment:
Tests can also be `skipped` during `dbt build`, e.g. if it's testing a model that errored

Reply:
After you update for Jeremy's comment above, this should be complete!

Reply:
good point! donezo